### PR TITLE
Add operation details

### PR DESCRIPTION
**Pull request by trentkrogers**
### 📝 Description

Adds osmosis operation details

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/16307439/183106841-83ab1359-7477-4186-b73c-ff7b1069d2f8.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by marco-figment — APPROVED** [review]
Nice!
i shipit

File: apps/ledger-live-mobile/src/families/osmosis/operationDetails.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+import { getAccountUnit } from "@ledgerhq/live-common/account/helpers";
+
+import type { CosmosDelegationInfo } from "@ledgerhq/live-common/families/cosmos/types";
+import { useSelector } from "react-redux";
+import Section from "../../screens/OperationDetails/Section";
+import { urls } from "../../config/urls";
+import { discreetModeSelector, localeSelector } from "../../reducers/settings";
+
+function getURLFeesInfo(op: Operation): ?string {
+  return op.fee.gt(200000) ? urls.cosmosStakingRewards : undefined;
```
Comment (JesseKuntz) [inline apps/ledger-live-mobile/src/families/osmosis/operationDetails.js:22]: Maybe:

```suggestion
  return op.fee.gt(200000) ? urls.osmosisStakingRewards : undefined;
```

File: apps/ledger-live-mobile/src/families/osmosis/operationDetails.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+function getURLWhatIsThis(op: Operation): ?string {
+  return op.type !== "IN" && op.type !== "OUT"
+    ? urls.cosmosStakingRewards
```
Comment (JesseKuntz) [inline apps/ledger-live-mobile/src/families/osmosis/operationDetails.js:27]: Same as above ⬆️

**Review by JesseKuntz — COMMENTED** [review]
Nice work! Maybe a couple URLs that should be osmosis instead? Not sure.

File: apps/ledger-live-mobile/src/families/osmosis/operationDetails.js (hunk 1 above)
Comment (trentkrogers) [reply apps/ledger-live-mobile/src/families/osmosis/operationDetails.js:22]: Nice catch! I thought I had checked these over but clearly not closely enough.

**Pull request merged**